app/renderer/components/lobby/modals/SessionSettings.tsx

```diff
@@ -19,6 +19,9 @@ import Dialog, {
   DialogTitle
 } from 'material-ui/Dialog'
 import { IReactReduxProps } from 'types/redux-thunk'
+import { Dropdown } from 'renderer/components/settings/controls';
+import { setSessionData } from 'renderer/lobby/actions/session';
+import { getMaxUsers } from 'renderer/lobby/reducers/session';
```

Comment:
Prettier should clean up the semi-colons when you save this file again. I need to add a Git pre-commit hook to take care of this automatically.